README.md

```diff
@@ -1,46 +1,62 @@
 
-### Take a look of addon examples
-#### Drawer & Toggle
-![drawer](./assets/drawer-toggle.png)
+## Getting Started
+### Build and Serve Plugin Locally
+_Note: Addons are sometimes referred to as plugins in the code_
+1. Either clone this whole repo, or clone a specific example plugin directory. More information about each example plugin can be found below.
+2. In the root directory of your chosen example plugin, install the dependencies using `npm install`
+3. Serve the plugin locally using `npm run start`. This will serve the built plugin at http://localhost:9100/plugin.js
 
-#### In-call view
-![in-call](./assets/in-call-view.png)
+### Using Your Plugin
+#### Internal (Coviu) Developers
+* [Plugin Development Steps](https://coviu.atlassian.net/wiki/spaces/TD/pages/285376535/Developing+a+new+plugin#Testing-the-plugin-locally)
+#### External Developers
+* [Coviu Add-on Documentation: Plugin Development Steps](https://coviu.readme.io/docs/technical-documentation#development-steps)
 
-#### Pre-call & Post-call views (phases before entering the call and after leaving the call)
-![pre-call-post-call](./assets/pre-call-post-call-views.png)
+Refer to Coviu's official documentation for external developers. The page linked above above details how to set up a development environment for working on locally served plugins.
 
-### Build and Serve Plugin locally
-_Note: Addons are sometimes referred to as plugins in the code_
-1. Clone one example from [Coviu Demo Plugin](https://github.com/coviu/coviu-addons)
-2. Install the dependencies using `npm install`
-4. Serve the plugin locally using `npm run start`. This will serve the built plugin at http://localhost:9100/plugin.js
+## Example Plugins
+### 1. In-call Data Stream Example
+A datachannel stream (datastream) is a stream established between two call participants that allows for the exchange of raw byte data. It has no additional handing for the data it carries, there is no history mechanism in place, and intended for passing data between two participants as quickly as possible.
+
+This example demonstrates in-call data streaming using a button which streams data to all other participants when clicked.
+
+![data-streaming](./assets/in-call-data-streaming.png)
+
+### 2. In-call Mesh Syncing Example
+This example demonstrates using the call mesh. A synchronized state is maintained for all the plugins during the call. This is done using CRDT data-structures. A plugin on any peer can push state changes to the webRTC mesh causing those changes to propagate to all the peers. This is stored in resource.metadata
+
+See the plugin API function api.resource.transaction() for how this is done.
+
+This example demonstrates in-call mesh syncing using a text input which is editable by all participants of the call, and will update when any participant makes a change.
 
-### Register for Staging access
```

Comment:
These steps are still relevant (Register for Staging access onward), can you include them in the updated readme.

Reply:
Those steps are covered in our official technical documentation which is already linked in this README (https://coviu.readme.io/docs/technical-documentation#development-steps). The 'Using Your Plugin' step of this README should point developers to the correct documentation for getting their plugins running in a call regardless of whether they're Coviu devs, or external devs.